ml.py

```diff
@@ -259,6 +279,14 @@ def __init__(self, base_reg, cv=5, groups=None, verbose=False, n_bins_stratify=N
 
         
     def fit(self, X, y, **kwargs):
+        # Apply transformation here if True
+        if self.add_box_cox_target:
+            pt = PowerTransformer(method="box-cox")
```

Comment:
Just as a comment, since Power Transformer is used here, it makes more sense to use its own transform method instead of using another library from scipy for that task. But that's just a design concern. Also, would be good to eventually add the transform as an internal object or method. That would allow reversing predictions that are coming directly from base_reg.